examples/src/bin/blinky_erased_pins.rs

```diff
@@ -2,38 +2,41 @@
 //!
```

Comment:
This suggestion for line 1 of this file
```suggestion
//! Blinks 3 LEDs
```

Also, not sure what the goal of setting a button pin that prints "button" in this example? I think we should mention that in the description or remove it.

Reply:
We should mention it since it does show how to have a function which takes any input pin. What we do with the input isn't important